protocols/v1/src/error.rs

```diff
@@ -2,14 +2,15 @@ use crate::methods::{Method, MethodError};
 
 #[derive(Debug)]
 #[non_exhaustive]
-pub enum Error {
+pub enum Error<'a> {
     // Previously expected HexBytes to never exceed 32 bytes in length, but when used with
```

Comment:
Now that HexBytes is gone can this comment block be removed in favor of a comment about BadU256Convert?